activities.el

```diff
@@ -881,13 +889,18 @@ with prefix argument, choose another activity."
 
 (defun activities-bookmark-store (activity)
   "Store a `bookmark' record for ACTIVITY."
-  (bookmark-maybe-load-default-file)
   (let* ((activities-name (activities-activity-name activity))
          (bookmark-name (concat activities-bookmark-name-prefix activities-name))
          (props `((activities-name . ,activities-name)
                   (handler . activities-bookmark-handler))))
     (bookmark-store bookmark-name props nil)))
 
+(defun activities-bookmark-delete (activity)
+  "Delete the `bookmark' record for ACTIVITY."
+  (let* ((activities-name (activities-activity-name activity))
+         (bookmark-name (concat activities-bookmark-name-prefix activities-name)))
+    (bookmark-delete bookmark-name)))
```

Comment:
It would be a bit more robust to find the bookmark to delete by finding ones that have the properties matching the ones made by the store function.  That way it will still work even if the bookmark has been renamed.